README.md

```diff
@@ -100,7 +125,11 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema describes the structure (column names, data type {string (text/varchar), integer, boolean}, any constraints) of a table in a database. It both instructs the database on what information to store and where, and informs the users/programmers what type of data they can expect in a table.
```

Comment:
Just to clarify, a schema describes the structure of the database which includes the table(s) and so totally correct that it describes the tables. I like [this analogy](http://stackoverflow.com/a/298751)